src/Domain/DomainCreators/Sphere.cpp

```diff
@@ -0,0 +1,112 @@
+                                                   with_equiangular_map_)),
+          std::make_unique<Wedge3DType>(Wedge3DMap(inner_radius_, outer_radius_,
+                                                   Direction<3>::lower_eta(), 0,
+                                                   with_equiangular_map_)),
+          std::make_unique<Wedge3DType>(Wedge3DMap(inner_radius_, outer_radius_,
+                                                   Direction<3>::upper_xi(), 0,
+                                                   with_equiangular_map_)),
+          std::make_unique<Wedge3DType>(Wedge3DMap(inner_radius_, outer_radius_,
+                                                   Direction<3>::lower_xi(), 0,
+                                                   with_equiangular_map_)),
+          std::make_unique<
+              CoordinateMap<Frame::Logical, Frame::Inertial, AffineMap3D>>(
+              AffineMap3D{AffineMap(-1.0, 1.0, -1.0 * inner_radius_ / sqrt(3.0),
+                                    inner_radius_ / sqrt(3.0)),
+                          AffineMap(-1.0, 1.0, -1.0 * inner_radius_ / sqrt(3.0),
+                                    inner_radius_ / sqrt(3.0)),
+                          AffineMap(-1.0, 1.0, -1.0 * inner_radius_ / sqrt(3.0),
+                                    inner_radius_ / sqrt(3.0))})),
+      corners};
+}
+
+std::vector<std::array<size_t, 3>> Sphere::initial_extents() const noexcept {
+  return {
+      {{initial_number_of_grid_points_[1], initial_number_of_grid_points_[1],
+        initial_number_of_grid_points_[0]}},
+      {{initial_number_of_grid_points_[1], initial_number_of_grid_points_[1],
+        initial_number_of_grid_points_[0]}},
+      {{initial_number_of_grid_points_[1], initial_number_of_grid_points_[1],
+        initial_number_of_grid_points_[0]}},
+      {{initial_number_of_grid_points_[1], initial_number_of_grid_points_[1],
+        initial_number_of_grid_points_[0]}},
+      {{initial_number_of_grid_points_[1], initial_number_of_grid_points_[1],
+        initial_number_of_grid_points_[0]}},
+      {{initial_number_of_grid_points_[1], initial_number_of_grid_points_[1],
+        initial_number_of_grid_points_[0]}},
+      {{initial_number_of_grid_points_[1], initial_number_of_grid_points_[1],
+        initial_number_of_grid_points_[1]}}};
+}
+std::vector<std::array<size_t, 3>> Sphere::initial_refinement_levels() const
+    noexcept {
+  return {{{initial_refinement_, initial_refinement_, initial_refinement_}},
+          {{initial_refinement_, initial_refinement_, initial_refinement_}},
+          {{initial_refinement_, initial_refinement_, initial_refinement_}},
+          {{initial_refinement_, initial_refinement_, initial_refinement_}},
+          {{initial_refinement_, initial_refinement_, initial_refinement_}},
+          {{initial_refinement_, initial_refinement_, initial_refinement_}},
+          {{initial_refinement_, initial_refinement_, initial_refinement_}}};
```

Comment:
How about `{7, make_array<3>(initial_refinement_)}` (might need an explicit type for the vector)?